lib/dependencies/ImportDependency.js

```diff
@@ -14,14 +15,36 @@ const ModuleDependency = require("./ModuleDependency");
 class ImportDependency extends ModuleDependency {
 	constructor(request, block) {
 		super(request);
+
 		this.block = block;
 	}
 
 	get type() {
 		return "import()";
 	}
+
+	serialize(context) {
+		const { write } = context;
+
+		write(this.block);
+
+		super.serialize(context);
+	}
+
+	deserialize(context) {
+		const { read } = context;
+
+		this.block = read();
+
+		super.deserialize(context);
+	}
 }
 
+makeSerializable(
+	ImportDependency,
+	"webpack/lib/dependencies/ImportDependency "
```

Comment:
Extra trailing space here:
```suggestion
	"webpack/lib/dependencies/ImportDependency"
```